### PR TITLE
Books with actual pages

### PR DESCRIPTION
**Pull request by BillyBangles**
Adds a new object, book/bundle, which works the same way as a paper bundle except you can't take the pages out. You can create one by clipping some papers together and putting the bundle into the book binder. The binder won't take any bundles with photos in it because that won't play nice with the database.

I did this to fix the infamous "PART 1" "PART 2" "PART 3" etc. books from the library system, as well as books that just have a google drive link in them instead of the actual book. What real life book have you read that only has one page anyways?

### CODE REVIEW

**Comment by Wutnold124** [comment]
nice(?)

**Comment by BillyBangles** [comment]
@Anewbe You can merge if you like but you might wanna slap a Do Not Merge on here until I make sure you can upload these to the database.

**Comment by Anewbe** [comment]
We won't merge til you say it's ready, don't worry.

File: code/modules/library/lib_machines.dm [hunk 1]
```diff
@@ -457,17 +457,37 @@ datum/borrowbook // Datum used to keep track of who has borrowed what when and f
...
-		var/obj/item/weapon/book/b = new(src.loc)
-		b.dat = O:info
-		b.name = "Print Job #" + "[rand(100, 999)]"
-		b.icon_state = "book[rand(1,7)]"
-		qdel(O)
+	if(istype(O, /obj/item/weapon/paper) || istype(O, /obj/item/weapon/paper_bundle))
+		if(istype(O, /obj/item/weapon/paper))
+			user.drop_item()
+			O.loc = src
+			user.visible_message("[user] loads a sheet of paper into [src].", "You load a sheet of paper into [src].")
```
Comment (Belsima) [inline code/modules/library/lib_machines.dm]: \the

**Pull request closed without merging**